src/components/navbar/Navbar.jsx

```diff
@@ -16,20 +16,30 @@ const Navbar = () => (
         <li className="navbar__items navbar__items--name">
           <Link to="/">Home</Link>
         </li>
-        <li className="navbar__items navbar__items--name">Dashboard</li>
+        <li className="navbar__items navbar__items--name">
+          <Link to="/dashboard">Dashboard</Link>
+        </li>
         <li className="navbar__items navbar__items--name">
```

Comment:
You should either use all underscores or all hyphens while naming classnames. it is good writing practice.

Reply:
Yeah, you are right but I have used the BEM naming convention here. This is one of their format they recommend while naming classes.